docs/wallet-extension/wallet-extension.md

```diff
@@ -35,16 +35,20 @@ tools.
 
 ## Usage
 
-1. Compile the wallet extension binary by cloning the [Obscuro repository](https://github.com/obscuronet/go-obscuro)
-   and running the following command from the root of the repository:
+1. The wallet extension can be downloaded from the [Obscuro release page](`https://github.com/obscuronet/go-obscuro/releases`) 
+   where the binary is built for the Linux, MacOS and Windows platforms. Download the binary for the latest release and 
+   rename to `wallet_extension`. Alternatively the binary can be compiled by cloning the 
+   [Obscuro repository](https://github.com/obscuronet/go-obscuro) and running the following command from the root of 
```

Comment:
I thought there is an issue using wallet extensions which have been manually compiled?

Reply:
I'm not aware of that - do you know what the problem is? You do have to go through the opening as a trusted program on MacOS route, which is painful, but otherwise my tests run fine through it when you download from the releases page.

Reply:
If it is the unsigned mac app issue, I can make this more clear in the document ... ideally we would need to get a developer ID from apply so we can sign these presumably.

Reply:
OK, lets see what the feedback is. Ok for now.